gui-wm/hikari/hikari-2.1.1.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="wayland compositor inspired by CWM"
+HOMEPAGE="https://hikari.acmelabs.space/"
+SRC_URI="https://hikari.acmelabs.space/releases/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="gamma layershell screencopy suid virtual +X +man"
+
+DEPEND="
+	dev-libs/libinput:=
+	dev-libs/libucl
+	>=gui-libs/wlroots-0.11.0
+	media-libs/libglvnd
+	x11-libs/cairo:=[X,svg]
+	x11-libs/pango:=[X]
+	x11-libs/pixman
+	x11-libs/libxkbcommon:=[X]
+	sys-libs/pam
+"
+
+RDEPEND="
+	${DEPEND}
+	x11-misc/xkeyboard-config
+"
+
+BDEPEND="
+	${DEPEND}
+	virtual/pkgconfig
+	dev-libs/wayland-protocols
+"
+
+# keep this as others OS's are using this as reference
+PATCHES=(
+	"${FILESDIR}/${P}-gnu-make.patch"
```

Comment:
What's up with this patch? It's huge. Where does it come from and why it's needed?

Reply:
hikari is written to use `bmake` instead of GNUMake and bmake is not supported by the eutils class and it is `~ARCH` for all arches.
This patch moves the build system to GNU Make.
Its also used by people on the official hikari channel and everyone over there uses this gentoo patch to use gmake, so I'd like to keep it here.
I've also added the steps in conversion at the top of this patch so it shouldn't be a problem.